py-polars/polars/dependencies.py

```diff
@@ -246,4 +250,5 @@ def _check_for_pydantic(obj: Any) -> bool:
     "_PANDAS_AVAILABLE",
     "_PYARROW_AVAILABLE",
     "_ZONEINFO_AVAILABLE",
+    "_GEVENT_AVAILABLE",
```

Comment:
Might want to move this up to line 248 to keep list (section) sorted alphabetically.